fw/vhost.c

```diff
@@ -132,6 +132,11 @@ static const TfwCfgEnum tfw_method_enum[] = {
  */
 #define TFW_CAPUACL_ARRAY_SZ	(32)
 
+
+#define TFW_FRANG_HTTP_METHODS_MASK_DEFAULT ((1 << TFW_HTTP_METH_GET) | \
+		(1 << TFW_HTTP_METH_HEAD) | \
+		(1 << TFW_HTTP_METH_POST))
+
```

Comment:
```suggestion
#define TFW_FRANG_HTTP_METHODS_MASK_DEFAULT ((1 << TFW_HTTP_METH_GET) | \
                                             (1 << TFW_HTTP_METH_HEAD) | \
                                             (1 << TFW_HTTP_METH_POST))   

```
We almost never use 2 empty lines in C/C++ code and we use alignments (tabs plus spaces)